packages/cli/go.mod

```diff
@@ -26,21 +26,54 @@ require (
 	github.com/fatih/color v1.12.0
 	github.com/golang/mock v1.6.0
 	github.com/jeremywohl/flatten v1.0.1
-	github.com/kr/pretty v0.2.1 // indirect
 	github.com/rs/zerolog v1.22.0
 	github.com/rsc/wes_client v0.0.0-00010101000000-000000000000
 	github.com/spf13/afero v1.6.0
 	github.com/spf13/cobra v1.2.1
 	github.com/spf13/pflag v1.0.5
-	github.com/stretchr/objx v0.2.0 // indirect
 	github.com/stretchr/testify v1.7.0
 	github.com/xeipuuv/gojsonschema v1.2.0
-	golang.org/x/net v0.0.0-20211015210444-4f30a5c0130f
-	golang.org/x/sys v0.0.0-20211019181941-9d821ace8654 // indirect
-	golang.org/x/text v0.3.7 // indirect
-	golang.org/x/tools v0.1.10 // indirect
-	gopkg.in/check.v1 v1.0.0-20190902080502-41f04d3bba15 // indirect
+	golang.org/x/net v0.1.0
 	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b
 )
 
+require (
+	github.com/VividCortex/ewma v1.1.1 // indirect
```

Comment:
Out of curiosity, why so many new indirect dependencies?  Were these added automatically?

Reply:
Yes, they were added automatically.